reviewrot.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+import yaml
+import logging
+import logging.config
+from reviewrot import get_git_service
+from os.path import expanduser
+
+def main(fltr, value, duration):
+    """
+    starts execution
+    """
+    #TODO: Implement StreamHandler
+    # logging.config.fileConfig('log.conf')
+    log = logging.getLogger(__name__)
```

Comment:
When defining a logger outside of a library (such as in this entry point script), it's best not to use `__name__` since it will not be the value you might expect. In this scenario just use "reviewrot".